Doc 3078 Database section of RC - updates from Doc Bash (#2922)
RedisLabs/redislabs-docs: content/rc/databases/migrate-databases.md

The source database remains active while the data migrates.

> Two processes for syncing are outlined as examples in the next sections.

Reviewer: ```suggestion
The following sections describe Active-Passive syncing processes.
```

> ### General process for Active Passive syncing 

Reviewer: ```suggestion
### General Active-Passive syncing process
```
OR
```suggestion
### General Active-Passive syncing
```

> To do this, specify the target database as an Active-Passive replica of the source database.  The general process is:

Reviewer: ```suggestion
To migrate data using Active-Passive syncing, specify the target database as an Active-Passive replica of the source database:
```

> 1.  Get the public endpoint of the source database. These are available from the database list and the **General** section of the **Configuration** tab for the source database.

Reviewer: ```suggestion
1.  Get the source database's public endpoint from the **General** section of the database's **Configuration** tab.
```

An error will appear when syncing the two databases if the source and target databases are hosted on different Redis Cloud accounts. [Contact support](https://redis.com/company/support/) if you want to migrate a database between accounts using Active-Passive.
{{< /note >}}

> ### Detailed process for Active Passive syncing: databases on same account

Reviewer: This header is a bit long. Preferably it should fit on a single line.
```suggestion
### Detailed Active-Passive syncing process
```
OR
```suggestion
### Detailed Active-Passive syncing
```
OR
```suggestion
### Sync databases on same account
```

> Here's how this works for databases hosted on the same Redis Cloud account:

Reviewer: ```suggestion
If the databases belong to the same Redis Cloud account, follow these detailed steps to migrate data using Active-Passive syncing:
```

    {{<image filename="images/rc/migrate-data-target-edit.png" alt="Use the **Edit Database** button to change the configuration of the target database." >}}{{< /image >}}

> 5.  In the **Durability** section, enable **Active Passive Redis** and then select **Add URI**.

Reviewer: ```suggestion
5.  In the **Durability** section, enable **Active-Passive Redis** and then select **Add URI**.
```
